scripts/epa/airdata/EPA_AQI.tmcf

```diff
@@ -4,7 +4,7 @@ typeOf: dcs:StatVarObservation
 variableMeasured: dcs:AirQualityIndex_AirPollutant
 observationDate: C:EPA_AQI->Date
 observationAbout: C:EPA_AQI->Place
-observationPeriod: dcs:"P1D"
+observationPeriod: dcs:P1D
```

Comment:
it is just "P1D" (rangeIncludes of observationPeriod is Text - https://datacommons.org/browser/observationPeriod)

Reply:
thanks for the explanation - this makes sense :) updated